geometry_msgs/PoseStamped obj_pose
---
geometry_msgs/PoseStamped obj_pose